api/volume.go

```diff
@@ -127,6 +129,18 @@ func (s *Server) VolumeCreate(rw http.ResponseWriter, req *http.Request) error {
 		}
 	}
 
+	if volume.AccessMode == "" && volume.FromBackup != "" {
+		_, volumeName, _, err := backupstore.DecodeBackupURL(volume.FromBackup)
+		if err != nil {
+			return fmt.Errorf("failed to get backup and volume through URL: %v", err)
```

Comment:
```suggestion
			return errors.Wrapf(err, "failed to decode backup url %v", volume.FromBackup)
```

Reply:
Thank you. I will follow the suggestion to wrap up the error.